src/os_posix/os_fs.c

```diff
@@ -341,16 +342,19 @@ __posix_file_read(WT_FILE_HANDLE *file_handle,
 	    len >= S2C(session)->buffer_alignment &&
 	    len % S2C(session)->buffer_alignment == 0));
 
+	WT_STAT_FAST_CONN_INCR(session, block_active_read);
 	/* Break reads larger than 1GB into 1GB chunks. */
 	for (addr = buf; len > 0; addr += nr, len -= (size_t)nr, offset += nr) {
 		chunk = WT_MIN(len, WT_GIGABYTE);
+		WT_STAT_FAST_CONN_DECR(session, block_active_write);
```

Comment:
This looks like it is a leftover.  This function is read, and that is the write stat.